Userland/Libraries/LibWeb/WebDriver/Client.cpp

```diff
@@ -72,6 +72,7 @@ static constexpr auto s_webdriver_endpoints = Array {
     ROUTE(POST, "/session/:session_id/window/fullscreen"sv, fullscreen_window),
     ROUTE(POST, "/session/:session_id/element"sv, find_element),
     ROUTE(POST, "/session/:session_id/elements"sv, find_elements),
+    ROUTE(GET, "/session/:session_id/element/:element_id/computedrole"sv, get_computed_role),
```

Comment:
So far every method declaration and definition is in spec order. I'd really like to keep it that way. So this belongs just below the `is_element_enabled` line.

Reply:
Gotcha, I had thought it was alphabetical